src/lock_file/update.rs

```diff
@@ -483,6 +498,10 @@ impl<'p> LockFileDerivedData<'p> {
 
         Ok(locked_env
             .conda_packages(platform)
+            .map(|packages| {
+                packages
+                    .filter(|p| !no_path_dependencies || matches!(p.location(), UrlOrPath::Url(_)))
```

Comment:
Will this exclude local 'built' conda packages? intent is to exclude source packages.

Reply:
I guess it would be better to do `p.is_source().is_some()`

Reply:
Done @tdejager . Does something similar exist for Pypi records? or are path pypi record always source ?

What do you think of the naming of the flag? I did not want to use 'source' there, as some remote pypi packages can be source packages...

Reply:
No, unfortunately not yet, and a path could be a built dist. I'd guess you would need to check the extension if it is a wheel file..